app/views/qualifications/show.html.erb

```diff
@@ -14,7 +14,7 @@
         <h3 class="govuk-heading-m">Your details</h3>
         <p>
           Name on certificates<br /> 
-          <strong><%= @user.name.full %></strong>
+          <strong><%= [@teacher.first_name, @teacher.last_name].join(" ") %></strong>
         </p>
```

Comment:
I assume the user name just above the h1 should remain the one we get from Identity?